Tutorials/Multisite-Setup/index.md

```diff
@@ -0,0 +1,55 @@
+---
+versionFrom: 8.0.0
+meta.Title: "Multisite setup in Umbraco"
+product: "CMS"
+meta.Description: "A guide to multisite setup in Umbraco"
+---
+
+# Multisite setup on Umbraco Cloud
+
+This tutorial explains how to host multiple sites from one project/installation of Umbraco.
+For practical reasons, we recommend using [Baselines](../../Umbraco-Cloud/Getting-Started/Baselines/) on the Cloud projects.
+If you are planning to create a multilanguage site in Umbraco 8, please take a look at the [Multilanguage Setup](../Multilanguage-Setup/) tutorial instead.
+
+## Structuring your website
+
+The best way to handle a multisite solution is to have multiple root nodes in the Content section, where each root node would act as a separate website.
+
+Keep in mind all the websites in your solution will be using the same schema - meaning, in most cases, your content pages on website A will be using the same properties as on website B. 
+
+## Binding the hostnames to the project
+
+Before you can map your hostnames to individual websites in the solution, you should add them in the Hostnames section on the Cloud portal to ensure they are secured with TLS.
+
+![Adding hostnames to the project](images/1-addinghostnames.png)
+
+Keep in mind the [hostnames have to be configured in a specific way.](../../Umbraco-Cloud/Set-Up/Manage-Hostnames/).
+
+## Mapping the hostnames to individual websites/root nodes
```

Comment:
How about using a list format for the steps outlined in this section?

Reply:
done